### ISSUE TITLE
Quoted-Printable encoding not handled

### ISSUE THREAD

**Issue by alpha14** [cozy/cozy-contacts#183]
My android contact app use Quoted-Printable encoding for contacts with accents in their names (vCard 2.1)
The cozy app doesn't decode the content when displaying the contact name.
Here is an example:
```
BEGIN:VCARD
VERSION:2.1
N;CHARSET=UTF-8;ENCODING=QUOTED-PRINTABLE:=54=65=73=74;=53=C3=A9=62=61=73=74=69=65=6E;;;
FN;CHARSET=UTF-8;ENCODING=QUOTED-PRINTABLE:=53=C3=A9=62=61=73=74=69=65=6E=20=54=65=73=74
TEL;CELL:0699999999
END:VCARD
```

**Comment by alpha14** [comment]
It also seems that the cozy app on android doesn't sync contacts with Quoted-Printable encoding.

**Comment by clochix** [comment]
Thanks for reporting @alpha14.
@m4dz could you have a look at the Contact part ?
@jacquarg shall we open an issue on mobile repo ?

**Comment by jacquarg** [comment]
I just checked, and I have contacts with accents (and Quoted-Printable in VCard serialisation) which are synchronized through the Android mobile app. So if the problem is confirmed, it more specific.

**Comment by m4dz** [comment]
@clochix it seems the bug is more related to [cozy-vcard](https://github.com/cozy/cozy-vcard) rather than the Contacts nor the Mobile app. Maybe we should move this issue to it?

**Comment by clochix** [comment]
:-S
I just parsed above file with Cozy-vcard, and here's the result:
```
{
  "datapoints": [
    {
      "name": "tel",
      "type": "cell",
      "value": "0699999999"
    }
  ],
  "n": "Test;Sébastien;;;",
  "fn": "Sébastien Test"
}
```
We need to investigate.

**Comment by clochix** [comment]
The libraries used by cozy-vcard to handle quoted printable were missing on client side.

**Comment by clochix** [comment]
Hi @alpha14,

The problem with quoted printable should have been fixed in 2.1.2, could you give it a try?

**Comment by alpha14** [comment]
Hi @clochix,
Thanks, it works great now !
